sample/src/main/res/layout/activity_main.xml

```diff
@@ -12,36 +12,83 @@
         tools:context=".MainActivity">
 
     <EditText
-            android:id="@+id/channelText"
-            android:layout_width="match_parent"
-            android:layout_height="wrap_content"
             android:hint="@string/hint_channel"
+            android:id="@+id/channelText"
             android:inputType="text"
+            android:layout_height="wrap_content"
+            android:layout_width="match_parent"
             tools:ignore="Autofill"/>
 
     <EditText
-            android:id="@+id/logText"
             android:hint="@string/hint_log"
-            android:layout_width="match_parent"
-            android:layout_height="wrap_content"
+            android:id="@+id/logText"
             android:inputType="text"
+            android:layout_height="wrap_content"
+            android:layout_width="match_parent"
             tools:ignore="Autofill"/>
 
     <Button
             android:id="@+id/send"
-            style="?android:textAppearanceSmall"
-            android:layout_width="match_parent"
             android:layout_height="wrap_content"
             android:layout_marginTop="16dp"
+            android:layout_width="match_parent"
             android:text="@string/send_log"
-            android:textStyle="bold"/>
+            android:textStyle="bold"
+            style="?android:textAppearanceSmall"/>
+
     <Button
             android:id="@+id/tick"
-            style="?android:textAppearanceSmall"
-            android:layout_width="match_parent"
             android:layout_height="wrap_content"
             android:layout_marginTop="16dp"
+            android:layout_width="match_parent"
             android:text="@string/start_tick"
-            android:textStyle="bold"/>
+            android:textStyle="bold"
+            style="?android:textAppearanceSmall"/>
+
+    <Button
```

Comment:
Maybe can use other component (Spinner or others) to only switch one type at once.